e2e-scripts/sippy-e2e-sippy-e2e-setup-commands.sh

```diff
@@ -185,12 +185,12 @@ spec:
         imagePullPolicy: ${SIPPY_IMAGE_PULL_POLICY:-Always}
         resources:
           limits:
-            memory: 1G
+            memory: 3G
```

Comment:
There was evidence we were OOMing on the first attempt and having to retry to get through the import.